guides/common/modules/con_host-group-structures.adoc

```diff
@@ -0,0 +1,37 @@
+= Host group structures
+
+The fact that host groups can be nested to inherit parameters from each other allows for designing host group hierarchies that fit particular workflows.
+A well planned host group structure can help to simplify the maintenance of host settings.
+This section outlines four approaches to organizing host groups.
+
+[[figu-Life_Cycle_Environment_Based_Structure]]
+.Host group structuring examples
+
+image::host-group-structures-satellite.png[Host group structuring examples]
+
+[[brid-Flat_Structure]]
+*Flat structure*
```

Comment:
```suggestion
Flat structure::
```

The asciidoc syntax is kinda _unique_. I suggest to use the glossary things `::`, or, alternatively, `.Flat structure`.

Reply:
I applaud your use of the word _unique_ in this context. More importantly, updated to use description list markup (`::`).